src/EnergyPlus/BaseboardRadiator.cc

```diff
@@ -638,6 +636,8 @@ namespace BaseboardRadiator {
                             state, cCMO_BBRadiator_Water, this->EquipID, "User-Specified Maximum Water Flow Rate [m3/s]", this->WaterVolFlowRateMax);
                     }
                 } else {
+                    auto &zoneEqSizing = state.dataSize->ZoneEqSizing(state.dataSize->CurZoneEqNum);
+                    auto const &finalZoneSizing = state.dataSize->FinalZoneSizing(state.dataSize->CurZoneEqNum);
```

Comment:
I wonder why I keep missing these allocation issues. Even here this doesn't look right. At this else, the flow could be autosized and a zone sizing run might not be requested. I think there needs to be a CheckZoneSizing call here to fatal out before the sizing arrays are accessed.

    if (!FlowAutoSize && !state.dataSize->ZoneSizingRunDone) {
    } else {
        std::string_view const CompType = cCMO_BBRadiator_Water;
        std::string_view const CompName = this->EquipID;
        CheckZoneSizing(state, CompType, CompName);
        // now OK to access sizing arrays
        auto &zoneEqSizing = state.dataSize->ZoneEqSizing(state.dataSize->CurZoneEqNum);
        auto const &finalZoneSizing = state.dataSize->FinalZoneSizing(state.dataSize->CurZoneEqNum);
Comment

    }

Reply:
Several `CheckZoneSizing` calls were removed in #10721 and consolidated in the precheck function which is now called `checkForZoneSizing`.

Reply:
Right. And these references still need to be located where sizing is requested so this is good.